clang/test/Driver/sycl-device-optimizations.cpp

```diff
@@ -1,11 +1,11 @@
 /// Check that optimizations for sycl device are enabled by default:
-// RUN:   %clang -### -fsycl %s 2>&1 \
+// RUN:   %clang -### -fsycl --offload-new-driver %s 2>&1 \
```

Comment:
nit: All test files with the the old driver option have "old model" in their name.
Test files with the new driver model can have their names updated too.

Reply:
I'd rather not shuffle the names around too much.  The idea is that the 'old model' will be going away once we make it the default, so the files renamed as 'old-model' can just be removed (as well as the existing tests that weren't converted to the new offloading model)

Reply:
I've thought about this a little, and I'm going to rename all of the tests that I'm not converting/adding to the new model to use 'old-model'.  This way, we know which ones will go away cleanly (have old-model in the name) and we don't have to worry about the comments.